### PR TITLE
Prepare 0.24 release

### PR DESCRIPTION
**Pull request by gjsjohnmurray**
This prepares the 0.24.0 release of the main extension together with 0.3.0 releases of the four bundled drivers.

### CODE REVIEW

**Comment by gjsjohnmurray** [comment]
@mtxr @george-james-software - using the test containers I have completed manual testing of the five new VSIXes this PR builds. Please review the PR. Once I have approval I will merge it and create the v0.24.0 release.

@mtxr when the release has competed building please download the 5 VSIX artifacts that will (hopefully) be attached to the release. Then publish them on Marketplace. The doc also needs to be published at https://vscode-sqltools.mteixeira.dev/ so that the changelog gets updated. Autumatic publishing of doc doesn't seem to be working at the moment.

**Review by george-james-software — APPROVED** [review]

**Pull request merged**
